lib/v2/domain-shared/page_command.dart

```diff
@@ -7,6 +7,12 @@ class ShowErrorMessage extends PageCommand {
   ShowErrorMessage(this.message);
 }
 
+class ProcessInviteLink extends PageCommand {
```

Comment:
I think this base class is only for base cases that are used in multiple places. 
There should be a pageCommand class thats unique to the App class. And thats where this should go.

Reply:
app_page_commands.dart